src/shared/condition.c

```diff
@@ -262,6 +263,44 @@ static int condition_test_kernel_version(Condition *c, char **env) {
         return true;
 }
 
+static int condition_test_osrelease(Condition *c, char **env) {
+        _cleanup_free_ char *key = NULL, *actual_value = NULL;
+        const char *parameter = c->parameter, *requested_value;
+        OrderOperator order;
+        int r;
+
+        assert(c);
+        assert(c->parameter);
+        assert(c->type == CONDITION_OS_RELEASE);
+
+        r = extract_first_word(&parameter, &key, "!<=>", 0);
```

Comment:
hmm, do we want to support escaping/quoting here maybe? i.e. should the flags param be non-null?

Reply:
No opinion either way - in which case would it be useful?

Reply:
quoting is allowed now